src/analysis/validation_with_arguments.Rmd

```diff
@@ -297,7 +297,19 @@ trivial_variable_plots(validation, 'env', 'openethereum')
 ```
 Trivial variable plots are interesting, after we have introduced varying program length. `openethereum` in particular has a good correlation there, which is caused by the large, constant cost of running an instruction (~250ns, regardless of OPCODE).
 
-Nevertheless, neither of the models is comparable with the model including the estimated `cost_ns`, see below.
+```{r}
```

Comment:
This is unrelated to the PR, a fixup to the validation script allowing us to better compare the "trivial" models